pages/checkin.tsx

```diff
@@ -168,20 +169,18 @@ export default function CheckinPage() {
           </Text>
           <Box height={1} />
           <Text>
+            <p>{introText}</p>
             <p>
-              {isCareEnv
-                ? 'Durch die aktuellen Corona-Verordnungen müssen Sie Ihre Kontaktdaten hinterlegen, wenn Sie in einem Betrieb sind der zu Schutzmaßnahmen verpflichtet ist, wie z.B Pflegeeinrichtungen. Die App kann auch freiwillig genutzt werden, um die Nachverfolgung zu unterstützen.'
-                : 'Durch die aktuellen Corona-Verordnungen musst du deine Kontaktdaten hinterlegen, wenn Du in einem Betrieb bist der zu Schutzmaßnahmen verpflichtet ist, wie z.B Restaurants. Die App kann auch freiwillig genutzt werden, um die Nachverfolgung zu unterstützen.'}
-            </p>
-            <p>
-              {isCareEnv
+              {isCareEnv || isFreseniusEnv
                 ? 'So kann das Gesundheitsamt Sie anrufen, wenn es notwendig ist.'
                 : 'So kann das Gesundheitsamt Dich anrufen, wenn es notwendig ist.'}
```

Comment:
Maybe we can try to avoid all these ternaries by putting all the values into `whitelabels.ts`

Reply:
Generally yes. In this case the text handles casual vs. more formal address of users. I think this should be handled by a setting in combination with I18n. I can change this so we have a setting for this in the whitelabels config. But I will not update the whole app to use this in this pr. We should do this when we implement a more solid way of handling whitelabels.